pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/basicpocogkehubfeaturemembership/dependencies.yaml

```diff
@@ -0,0 +1,111 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+kind: Project
+metadata:
+  annotations:
+    cnrm.cloud.google.com/deletion-policy: "abandon"
+  name: gkehubfm-${uniqueId}
+spec:
+  resourceID: ${TEST_DEPENDENT_ORG_PROJECT_ID}
+  name: "Dependent Project"
+  organizationRef:
+    external: ${TEST_ORG_ID}
+  billingAccountRef:
+    external: ${TEST_BILLING_ACCOUNT_ID}
+---
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: ${TEST_DEPENDENT_ORG_PROJECT_ID}
+    cnrm.cloud.google.com/deletion-policy: "abandon"
+  name: container.googleapis.com
+---
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerCluster
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: ${TEST_DEPENDENT_ORG_PROJECT_ID}
+  name: gke-basic-poco-${uniqueId}
+spec:
+  location: us-central1-a
+  initialNodeCount: 1
+  workloadIdentityConfig:
+    workloadPool: ${TEST_DEPENDENT_ORG_PROJECT_ID_WITHOUT_QUOTATION}.svc.id.goog
+---
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: ${TEST_DEPENDENT_ORG_PROJECT_ID}
+    cnrm.cloud.google.com/deletion-policy: "abandon"
+  name: anthos.googleapis.com
+---
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: ${TEST_DEPENDENT_ORG_PROJECT_ID}
+    # Abandon the resource to unblock cleanup; Otherwise it fails with error message
+    # "Error waiting for api to disable ... ensure there are no more resources managed
+    # by this service." Likely there is some delay in GKEHub resources clean up.
+    cnrm.cloud.google.com/deletion-policy: "abandon"
+  name: gkehub.googleapis.com
+---
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: ${TEST_DEPENDENT_ORG_PROJECT_ID}
+    cnrm.cloud.google.com/deletion-policy: "abandon"
+  name: anthosconfigmanagement.googleapis.com
+---
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: ${TEST_DEPENDENT_ORG_PROJECT_ID}
+    cnrm.cloud.google.com/deletion-policy: "abandon"
+  name: anthospolicycontroller.googleapis.com
+---
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
```

Comment:
I recall GKEHub features are singleton resources. Will there be another policycontroller feature created in the same project? If so are their configuration the same? I want to make sure there is no racing condition.

Reply:
Sorry I don't think I'm following here -- Joyce, would you mind clarifying here please 🙏🏼 

> Will there be another policycontroller feature created in the same project? If so are their configuration the same? 

> How about changing the test naming convention to reflect that, something like policycontroller.gkehubfeature.gkehub.cnrm.cloud.google.com

Reply:
By "singleton" resource, I meant the API only allows one POCO feature instance in one project. To have a POCO feature, the resourceID must be `policycontroller`. So basically basicpoco test and fullpoco test here are both pointing to the exact same underlying GKE feature instance.